docs/user/private-app-repository.md

```diff
@@ -101,6 +101,14 @@ To add Harbor as the private chart repository, go to `Configuration > App Reposi
 
 Once you create the repository you can click on the link for the specific repository and you will be able to deploy your own applications using Kubeapps.
 
+### Harbor: Authentication/Authorization
+
+It is possible to configure Harbor to use authentication:
+
+- Using HTTP [basic authentication](https://github.com/goharbor/harbor/blob/master/docs/user_guide.md#working-with-helm-cli). To use this feature:
```

Comment:
if using basic authentication is the only option you can remove the list. Also the link is pointing to `working-with-helm-cli` that doesn't seem related. Is that the link you wanted?

Reply:
That link points to the doc about how to use 'basic authentication' in Helm CLI to authenticate Harbor chart repository. It's similiar to `Using HTTP [basic authentication](https://chartmuseum.com/docs/#basic-auth)`. We can remove the link if not proper?
There should be other authentication ways for Harbor. We can test it and add here later. The doc above only shows Basci Auth.

Reply:
I see, those docs also talks about using a SSL cert which may be confusing, I would leave the link out because it's not really relevant in this context.

For the format I would merge this line with the above:

```
It is possible to configure Harbor to use authentication using HTTP basic authentication ...
```

Reply:
Thanks. This makes sense. Updated in new commit.